src/compiler/style/css-imports.ts

```diff
@@ -6,6 +6,19 @@ import { parseStyleDocs } from '../docs/style-docs';
 import { resolveModuleIdAsync } from '../sys/resolve/resolve-module-async';
 import { stripCssComments } from './style-utils';
 
+/**
+ * Parse CSS imports into an object which contains a manifest of imports and a
+ * stylesheet with all imports resolved and concatenated.
```

Comment:
as always, these and other comments represent my best effort to understand what's going on...could be missing something!